java/java.lsp.server/script/etc/nbcode.clusters

```diff
@@ -6,5 +6,6 @@ java
 groovy
 cpplite
 extra
+apisupport
```

Comment:
This looks like apisupport will be built into VSNetBeans by default with all existing included clusters?